js_modules/dagit/packages/core/src/assets/__stories__/LaunchAssetChoosePartitionsDialog.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import {MockedProvider} from '@apollo/client/testing';
+import {Meta} from '@storybook/react/types-6-0';
+import * as React from 'react';
+
+import {StorybookProvider} from '../../testing/StorybookProvider';
+import {LaunchAssetChoosePartitionsDialog} from '../LaunchAssetChoosePartitionsDialog';
+import {ReleasesJobProps} from '../__fixtures__/LaunchAssetChoosePartitionsDialog.fixtures';
+import {
+  ReleaseFiles,
+  ReleaseFilesMetadata,
+  ReleaseZips,
+  ReleasesMetadata,
+  ReleasesSummary,
+} from '../__fixtures__/PartitionHealthQuery.fixtures';
+import {NoRunningBackfills} from '../__fixtures__/RunningBackfillsNoticeQuery.fixture';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'LaunchAssetChoosePartitionsDialog',
+  component: LaunchAssetChoosePartitionsDialog,
+} as Meta;
+
+export const Empty = () => {
+  return (
+    <StorybookProvider>
```

Comment:
Does it need the whole `StorybookProvider`? It's a bit like `TestProvider` in that it provides the full mocked schema version of the Apollo provider, which it looks like you probably shouldn't need here.

Reply:
Ah okay, I wasn't sure if I needed it tbh